spec/localhost/sphinx_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe 'sphinx installation' do
+
+  describe command('/usr/local/sphinx-2.2.6/bin/indexer') do
```

Comment:
Is Sphinx a 'package'? If so, yes, ServerSpec has something. Otherwise unfortunately Sphinx doesn't respond to the customary -v --version request. It's is the only thing of everything we've tested that needs this type of probing...